services/storage/src/simcore_service_storage/api/_worker_tasks/_progress_utils.py

```diff
@@ -0,0 +1,24 @@
+from typing import Final
+
+from models_library.progress_bar import ProgressReport
+from pydantic import NonNegativeFloat
+from tqdm import tqdm
+
+TQDM_EXPORT_OPTIONS: Final[dict] = {
+    "unit": "",
+    "unit_scale": True,
+    "unit_divisor": 1024,
+    "colour": "yellow",
+    "miniters": 1,
+    "ncols": 100,
+}
+
+
+def get_tqdm_progress(total: NonNegativeFloat, *, description: str) -> tqdm:
+    return tqdm(**TQDM_EXPORT_OPTIONS, total=total, desc=description)
+
+
+def set_tqdm_absolute_progress(pbar: tqdm, report: ProgressReport) -> None:
```

Comment:
question. why you do not use ProgressBarData instead of tqdm here again?
I can show you this: https://github.com/ITISFoundation/osparc-simcore/pull/7375

Reply:
I think the object here is correct. Let's go though this one more time, since I'm not sure what you are trying to say here.